v3.2 only knows WL12XX_PLATFORM_DATA, newer kernels use WILINK_PLATFORM_DATA

Index: wl18xx/include/linux/wl12xx.h
===================================================================
--- wl18xx.orig/include/linux/wl12xx.h
+++ wl18xx/include/linux/wl12xx.h
@@ -62,7 +62,7 @@ struct wl12xx_platform_data {
 /* Platform does not support level trigger interrupts */
 #define WL12XX_PLATFORM_QUIRK_EDGE_IRQ	BIT(0)
 
-#ifdef CONFIG_WILINK_PLATFORM_DATA
+#if defined(CONFIG_WILINK_PLATFORM_DATA) || defined(CONFIG_WL12XX_PLATFORM_DATA)
 
 int wl12xx_set_platform_data(const struct wl12xx_platform_data *data);
 
